Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -234,6 +234,14 @@ extension RewardsCollectionViewController {
       rewardCell.delegate = self
     }
   }
+
+  override func collectionView(
```

Comment:
Just curious, why is this necessary?

Reply:
I wanted to make sure we remove the reference to the delegate when the cell gets dequeued. The reason I put it side by side `willDisplay` is for better readability but this could also go to `prepareForReuse`. It was entirely for better readability as we set the delegate when the cell gets displayed and `nil` it out when it leaves the screen.

I guess it's not really going to break anything (since the delegate is always `self` for any of these cells) in this use case but I wanted to use this properly...happy to remove.

Reply:
I'd say because all the cells have the same delegate there's no harm in leaving it - then you don't need to worry about re-adding it when the cell is configured.